### ISSUE TITLE
Installed header files should have C++ guards

### ISSUE THREAD

**Issue by thiagomacieira** [AVnu/libavtp#8]
They're missing:
```c
#ifdef __cplusplus
extern "C" {
#endif
....
#ifdef __cplusplus
}
#endif
```

So C++ source code can `#include` the headers.

**Comment by andrew-elder** [comment]
@thiagomacieira - thanks for the comments. Feel free to create a pull request with the changes you would like to see applied. I'm uncertain whether this repo is supposed to be "pure" C, or not. Comments anyone?

**Comment by aguedes** [comment]
Yes, this makes sense to me. I can fix it if @thiagomacieira isn't willing to.

**Comment by aguedes** [comment]
@thiagomacieira, PR https://github.com/AVnu/libavtp/pull/10 fixes this issue (PR merged already, by the way) so I believe we can close this issue.
